dev/com.ibm.ws.beanvalidation/src/com/ibm/ws/beanvalidation/OSGiBeanValidationImpl.java

```diff
@@ -397,6 +496,14 @@ private boolean isBeanValidationVersion11OrGreater() {
         return runtimeVersion.compareTo(BeanValidationRuntimeVersion.VERSION_1_1) >= 0 ? true : false;
     }
 
+    private boolean isBeanValidationVersion11() {
+        return runtimeVersion.compareTo(BeanValidationRuntimeVersion.VERSION_1_1) == 0 ? true : false;
```

Comment:
can remove the ternary operators `? true : false` here

Reply:
Done.